script_class.js

```diff
@@ -97,3 +97,51 @@ deleteBtn.forEach((btn) => {
     GetIndex();
   });
 });
+
+document.getElementById('date').innerHTML = Date();
+
+const list = document.getElementById('list-books');
+const subtitle = document.getElementById('subtitle');
+const form = document.getElementById('add-books');
+const contact = document.getElementById('contact');
+
+const btnList = document.getElementById('listLink');
+const btnAddNew = document.getElementById('addLink');
+const btnContact = document.getElementById('contactLink');
+
+function List() {
+  list.classList.remove('hide');
+  subtitle.classList.add('hide');
+  form.classList.add('hide');
+  contact.classList.add('hide');
+  btnList.classList.add('blue');
+  btnList.classList.remove('link_nav');
+  btnAddNew.classList.add('link_nav');
+  btnContact.classList.add('link_nav');
+}
```

Comment:
- [ ] When the app starts both the list and the form are shown at the same time while the navigation says I'm on the list section, kindly fix this issue so only the list shows when the app is started 👍 

![image](https://user-images.githubusercontent.com/50721479/204906881-ff82d77a-2380-46ac-b001-d08a8d57b64d.png)